ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLHttpService.java

```diff
@@ -398,14 +399,7 @@ private void handleGraphQLRequest(final RoutingContext routingContext) {
 
       final HttpServerResponse response = routingContext.response();
       vertx.executeBlocking(
-          future -> {
-            try {
-              final GraphQLResponse graphQLResponse = process(query, operationName, variables);
-              future.complete(graphQLResponse);
-            } catch (final Exception e) {
-              future.fail(e);
-            }
-          },
+          () -> process(query, operationName, variables),
```

Comment:
How would a failure in process be handled?

Reply:
Any exception fired in the context of the async execution results in a res.failed() in the result handler. I'm adding a test for this case though.